src/utils/localizedTypes.ts

```diff
@@ -4,6 +4,21 @@ import { sortBy } from 'lodash'
 import { components } from './apiSchema'
 import { tokenizeReference } from './tokenize'
 
+const extractCultivar = (scientificName: string): string | null => {
+  const cultivarIndex = scientificName.indexOf("'")
+  if (cultivarIndex === -1) {
+    return null
+  }
+  const m = scientificName.substring(cultivarIndex)
```

Comment:
This operates on the assumption that the cultivar name will always be the last thing in a scientific name, which may not always be true. Could the match be limited to the first and last occurrence of `'`? For example the greedy regex `/'(.*)'/`.